frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -1,15 +1,11 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import { LoadingInline, pluralize } from '@console/internal/components/utils';
 import { RedExclamationCircleIcon } from '@console/shared';
-
-const pluralizeWithNotation = (i: number, suffix: string, singular: string) =>
-  `${i}${suffix} ${i === 1 ? singular : `${singular}s`}`;
+import { LoadingInline, pluralize, humanizeNumber } from '@console/internal/components/utils';
 
 const formatCount = (count: number) => {
-  if (count < 1000) return pluralizeWithNotation(count, '', 'Object');
-  if (count >= 1000 && count < 1000000) return pluralizeWithNotation(count / 1000, 'K', 'Object');
-  return pluralizeWithNotation(count / 1000000, 'M', 'Object');
+  const hCount = humanizeNumber(count);
+  return `${hCount.string} Object${count === 1 ? '' : 's'}`;
```

Comment:
use `pluralize` instead of writing your own logic. 
```suggestion
  return `${hCount.string} ${pluralize(count, 'Object')}`
```

Reply:
@cloudbehl , won't be able to use `pluralize` funtion as it returns a string which has both count and the pluralized form of word.
`pluralize(8923423, 'Object') => '8923423 Objects'`

UI may look something like this,

![pluralize_disaster](https://user-images.githubusercontent.com/895234/62956798-2ec69b00-be11-11e9-9034-08a50f40dc5d.png)

Or, on a long shot, have to modify the `pluralize` function to accept a `humanize` function (as third optional parameter)...
`pluralize(8923423, 'Object', humanizeNumber) => '823.2m Objects'`

Reply:
Ack @aruniiird